plugins/profiles.koplugin/profile_1.lua

```diff
@@ -0,0 +1,14 @@
+local Device = require("device")
+local Screen = require("device").screen
```

Comment:
It doesn't make much practical difference but according to the coding style that should be:
```suggestion
local Screen = Device.screen
```